### ISSUE TITLE
Documentation Build Failing

### ISSUE THREAD

**Issue by kmdalton** [rs-station/reciprocalspaceship#275]
I created a new branch in which I did nothing but change the version number from 1.0.2 to 1.0.3. Creating a PR from this, #274, reveals that the documentation build is broken.

**Comment by kmdalton** [comment]
Downgrading the `sphinx` version to be `<8` fixed my build locally. 

Here's an example traceback from a local, failed build:

```python
# Sphinx version:   8.1.3
# Python version:   3.12.7 (CPython)
# Docutils version: 0.21.2
# Jinja2 version:   3.1.4
# Pygments version: 2.18.0

# Last messages:
#   developers/testing
#   
#   
#   reading sources... [ 62%]
#   developers/utilities
#   
#   
#   reading sources... [ 65%]
#   examples/0_quickstart
#   

# Loaded extensions:
#   sphinx.ext.mathjax (8.1.3)
#   alabaster (1.0.0)
#   sphinxcontrib.applehelp (2.0.0)
#   sphinxcontrib.devhelp (2.0.0)
#   sphinxcontrib.htmlhelp (2.1.0)
#   sphinxcontrib.serializinghtml (2.0.0)
#   sphinxcontrib.qthelp (2.0.0)
#   sphinx.ext.autodoc.preserve_defaults (8.1.3)
#   sphinx.ext.autodoc.type_comment (8.1.3)
#   sphinx.ext.autodoc.typehints (8.1.3)
#   sphinx.ext.autodoc (8.1.3)
#   sphinx.ext.autosummary (8.1.3)
#   sphinx.ext.napoleon (8.1.3)
#   sphinxcontrib.jquery (4.1)
#   sphinx_rtd_theme (unknown version)
#   nbsphinx (0.9.5)
#   sphinx.ext.linkcode (8.1.3)
#   sphinx_design (0.6.1)
#   sphinxcontrib.autoprogram (unknown version)
#   autodocsumm (8.1.3)

# Traceback:
Traceback (most recent call last):
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/sphinx/cmd/build.py", line 514, in build_main
    app.build(args.force_all, args.filenames)
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/sphinx/application.py", line 381, in build
    self.builder.build_update()
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/sphinx/builders/__init__.py", line 358, in build_update
    self.build(
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/sphinx/builders/__init__.py", line 385, in build
    updated_docnames = set(self.read())
                           ^^^^^^^^^^^
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/sphinx/builders/__init__.py", line 502, in read
    self._read_serial(docnames)
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/sphinx/builders/__init__.py", line 567, in _read_serial
    self.read_doc(docname)
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/sphinx/builders/__init__.py", line 630, in read_doc
    publisher.publish()
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/docutils/core.py", line 234, in publish
    self.document = self.reader.read(self.source, self.parser,
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/sphinx/io.py", line 106, in read
    self.parse()
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/docutils/readers/__init__.py", line 76, in parse
    self.parser.parse(self.input, document)
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/nbsphinx/__init__.py", line 667, in parse
    env.config.nbsphinx_prolog).render(env=env)
                                ^^^^^^^^^^^^^^^
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/home/kmdalton/opt/anaconda/envs/blah/lib/python3.12/site-packages/jinja2/environment.py", line 939, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 2, in top-level template code
TypeError: can only concatenate str (not "_StrPath") to str

```

**Comment by kmdalton** [comment]
resolved in #273
